flow/designs/asap7/mock-array/simulate.sh

```diff
@@ -27,6 +27,6 @@ verilator -Wall --cc \
   --exe \
   ../../../designs/src/mock-array/simulate.cpp
 
-make -C obj_dir -f VMockArray.mk
+make -j -C obj_dir -f VMockArray.mk
```

Comment:
`-j $(NUM_CORES)` would be more consistent with the rest of the flow and gives the control